deprecation-index.yml

```diff
@@ -268,3 +268,203 @@
+    Examples:
+'AssertLegacyTrait::assertOption()':
+    Rector: AssertOptionRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->optionExists() instead.'
+    Examples:
+'AssertLegacyTrait::assertOptionSelected()':
+    Rector: AssertOptionSelectedRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->optionExists() instead and check the "selected" attribute yourself.'
+    Examples:
+'AssertLegacyTrait::assertPattern()':
+    Rector: AssertPatternRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->responseMatches() instead.'
+    Examples:
+'AssertLegacyTrait::assertRaw()':
+    Rector: AssertRawRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->responseContains() instead.'
+    Examples:
+'AssertLegacyTrait::assert()':
+    Rector: AssertRector.php
+    PHPStan: 'Deprecated in drupal:8.0.0 and is removed from drupal:10.0.0. Use $this->assertTrue() instead.'
+    Examples:
+'AssertLegacyTrait::assertResponse()':
+    Rector: AssertResponseRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->statusCodeEquals() instead.'
+    Examples:
+'AssertLegacyTrait::assertText()':
+    Rector: AssertTextRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->responseContains() for non-HTML responses, like XML or Json. $this->assertSession()->pageTextContains() for HTML responses. Unlike the deprecated assertText(), the passed text should be HTML decoded, exactly as a human sees it in the browser.'
+    Examples:
+'AssertLegacyTrait::assertTitle()':
+    Rector: AssertTitleRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->titleEquals() instead.'
+    Examples:
+'AssertLegacyTrait::assertUniqueText()':
+    Rector: AssertUniqueTextRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->getSession()->pageTextContainsOnce() or $this->getSession()->pageTextMatchesCount() instead.'
+    Examples:
+'AssertLegacyTrait::assertUrl()':
+    Rector: AssertUrlRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->addressEquals() instead.'
+    Examples:
+'AssertLegacyTrait::buildXPathQuery()':
+    Rector: BuildXPathQueryRector.php
+    PHPStan: 'Deprecated in drupal:8.2.0 and is removed from drupal:10.0.0. Use $this->assertSession()->buildXPathQuery() instead.'
+    Examples:
+'AssertLegacyTrait::getAllOptions()':
+    Rector: GetAllOptionsRector.php
+    PHPStan: 'Deprecated in drupal:8.5.0 and is removed from drupal:10.0.0. Use $element->findAll('xpath', 'option') instead.'
```

Comment:
We have to double quote the string since it has single quotes in the deprecation message.

Reply:
@agentrickard made a fix in https://github.com/palantirnet/drupal-rector/pull/172, this is a PR to your branch so it'll update this PR